public/js/Histogram.js

```diff
@@ -12,21 +16,28 @@ _.mixin(require('lodash-deep'));
  * @param {array} featureNames - An array of the features used in this screen.
  */
 function Histogram(sampleData, element, featureNames) {
+    var self = this;
```

Comment:
Got it! Suggestion: copy this comment verbatim and add it somewhere in the code! =) Totally get it now.

It's not making me any more enamoured of anonymous functions, though... =P